static/src/stylesheets/module/facia/_container--video.scss

```diff
@@ -426,3 +426,16 @@ $video-width-desktop: 700px;
     bottom: 2px;
     color: $neutral-3;
 }
+
+.fc-item__video-container {
+    .youtube-media-atom {
+        z-index: 1;
+    }
+
+    .youtube-media-atom__iframe:not(.youtube__video-ready) {
+        ~ .youtube-media-atom__overlay {
+            visibility: hidden;
+            opacity: 0;       
```

Comment:
@GHaberis I removed the initial opacity transition from 0 to 1, the transition from 1 to 0 on play should still be there — if it doesn't make it too complex?